src/pages/workspace/WorkspaceMembersPage.js

```diff
@@ -148,7 +148,10 @@ class WorkspaceMembersPage extends React.Component {
 
         const canBeRemoved = this.props.policy.owner !== login && this.props.session.email !== login;
         if (!canBeRemoved) {
-            this.setState({showTooltipForLogin: login});
+            // Show growl notification to inform the user that they can not remove themselves.
+            this.setState({
+                showTooltipForLogin: login,
+            }, () => this.setState({showTooltipForLogin: ''}));
```

Comment:
I meant to say add a comment here to explain why we reset this immediately.

Reply:
Before I commit, are these changes ok?

```js
// We are resetting the state as a workaround of CheckboxWithTooltip bug.
this.setState({
    showTooltipForLogin: login,
}, () => this.setState({showTooltipForLogin: ''}));
```

Reply:
No, this is not useful. You can explain what resetting do here.